utils/sharding.py

```diff
@@ -0,0 +1,13 @@
+import os
+import hashlib
+
+
+SHARDS = int(os.environ.get('SHARDS', 1))
+SHARD_ID = int(os.environ.get('SHARD_ID', 0))
+
+
+def is_in_shard(value):
```

Comment:
would it be cheaper to return earlier if SHARDS is 1?

If we instrument an integration to support shards we will likely use them, so this won’t change anything, but if we ever switch back to one shard this is more correct.

Reply:
done